schema.js

```diff
@@ -0,0 +1,119 @@
+{
+    "_id": "getty_WA0682-001",
+    "source_name": "getty", 
+    "date_created_original": "1979-10-17", 
+    "date_created_at_source": "2005-08-31T00:00:00-07:00", 
+    "mediachain_record": null,
+    "title": [
+        "Male boxer practising using punch bag (blurred motion, toned B&W)"
+    ], 
+    "artist_names": [
+        "Mike Powell"
+    ], 
+    "attribution_list": [
+        {
+            "name": "Mike Powell",
+            "role": "artist", 
+            "details": null
+        }
+    ],
+    "provider_list": [
+        {
+            "name": "dpla"
+        }, 
+        {
+            "name": "Digital Library of Georgia"
+        }
+    ],
+    "sightings_list": [
+        {
+            "url_page": "http://digitalcollections.library.gsu.edu/cdm/ref/collection/ajc/id/8135", 
+            "url_image": "http://dlg.galileo.usg.edu/gsu/ajc/do-th:8135"
+        }
+    ], 
```

Comment:
Now have several fields which represent the single best / official sighting: `source`, `url_shown_at`, `url_direct`.

Also the image URLs represent only the single best sighting: `url_direct_cache`, `img_data`.